project/Dependencies.scala

```diff
@@ -107,8 +107,9 @@ object Dependencies {
       val codegen   = "org.hatdex" %% "slick-postgres-driver" % "0.1.2"
     }
 
-    val scalaGuice  = "net.codingwell" %% "scala-guice"  % "4.2.6"
-    val circeConfig = "io.circe"       %% "circe-config" % "0.8.0"
+    val scalaGuice  = "net.codingwell"     %% "scala-guice"  % "4.2.11"
+    val circeConfig = "io.circe"           %% "circe-config" % "0.8.0"
+    val janino      = "org.codehaus.janino" % "janino"       % "3.1.3"
```

Comment:
What is the janino? Where is it being used? I am asking because it's the first time I see it 😅

Reply:
http://janino-compiler.github.io/janino/ it's a compiler, but it's used for static analysis.

It *was* in build.sbt, but I moved It to deps for consistency.